data/simple/bergamot/outputs/int8shiftAlphaAll/ssse3/quality_estimator.expected

```diff
@@ -0,0 +1,20 @@
+[score Sentence]:0
+[words Scores]:Das(0.000) Bergamot-Projekt(0.000) wird(0.000) die(0.000) Übersetzung(0.000) der(0.000) Client-Seite(0.000) der(0.000) Maschine(0.000) in(0.000) einem(0.000) Webbrowser(0.000) ergänzen(0.000) und(0.000) verbessern.(0.000)
+
+[src Sentence]:Unlike current cloud-based options, running directly on users’ machines empowers citizens to preserve their privacy and increases the uptake of language technologies in Europe in various sectors that require confidentiality.
+[tgt Sentence]:Im Gegensatz zu aktuellen Cloud-basierten Optionen, die direkt auf den Rechnern der Nutzer laufen, die Bürgerinnen und Bürger, ihre Privatsphäre zu bewahren und erhöht die Verbreitung von Sprachtechnologien in Europa in verschiedenen Sektoren, die Vertraulichkeit erfordern.
+[score Sentence]:0.000
+[words Scores]:Im(0.000) Gegensatz(0.000) zu(0.000) aktuellen(0.000) Cloud-basierten(0.000) Optionen,(0.000) die(0.000) direkt(0.000) auf(0.000) den(0.000) Rechnern(0.000) der(0.000) Nutzer(0.000) laufen,(0.000) die(0.000) Bürgerinnen(0.000) und(0.000) Bürger,(0.000) ihre(0.000) Privatsphäre(0.000) zu(0.000) bewahren(0.000) und(0.000) erhöht(0.000) die(0.000) Verbreitung(0.000) von(0.000) Sprachtechnologien(0.000) in(0.000) Europa(0.000) in(0.000) verschiedenen(0.000) Sektoren,(0.000) die(0.000) Vertraulichkeit(0.000) erfordern.(0.000)
+
+[src Sentence]:Free software integrated with an open-source web browser, such as Mozilla Firefox, will enable bottom-up adoption by non-experts, resulting in cost savings for private and public sector users who would otherwise procure translation or operate monolingually.
+[tgt Sentence]:Freie Software, die mit einem Open-Source-Webbrowser wie Mozilla Firefox integriert ist, wird die Akzeptanz von unten nach oben durch Nicht-Experten ermöglichen, was zu Kosteneinsparungen für private und öffentliche Nutzer führt, die andernfalls Übersetzungen beschaffen oder einsprachig arbeiten würden.
+[score Sentence]:0.000
+[words Scores]:Freie(0.000) Software,(0.000) die(0.000) mit(0.000) einem(0.000) Open-Source-Webbrowser(0.000) wie(0.000) Mozilla(0.000) Firefox(0.000) integriert(0.000) ist,(0.000) wird(0.000) die(0.000) Akzeptanz(0.000) von(0.000) unten(0.000) nach(0.000) oben(0.000) durch(0.000) Nicht-Experten(0.000) ermöglichen,(0.000) was(0.000) zu(0.000) Kosteneinsparungen(0.000) für(0.000) private(0.000) und(0.000) öffentliche(0.000) Nutzer(0.000) führt,(0.000) die(0.000) andernfalls(0.000) Übersetzungen(0.000) beschaffen(0.000) oder(0.000) einsprachig(0.000) arbeiten(0.000) würden.(0.000)
+
+[src Sentence]:Bergamot is a consortium coordinated by the University of Edinburgh with partners Charles University in Prague, the University of Sheffield, University of Tartu, and Mozilla.
+[tgt Sentence]:Bergamot ist ein Konsortium, das von der Universität Edinburgh mit den Partnern Charles University in Prag, der University of Sheffield, der University of Tartu und Mozilla koordiniert wird.
+[score Sentence]:0.000
```

Comment:
The outputs are all 0, expecting some sensible values if you're checking in an actual working model.bin